src/scandeval/dataset_configs.py

```diff
@@ -784,6 +784,23 @@ def get_dataset_config(dataset_name: str) -> DatasetConfig:
     max_generated_tokens=5,
 )
 
+JENTOFT_CONFIG = DatasetConfig(
+    name="jentoft",
+    pretty_name="the Jentoft linguistic acceptability dataset",
+    huggingface_id="ScandEval/jentoft-la",
```

Comment:
Maybe standardise everything to 'jentoft' instead of using 'jentoft-la' sometimes?

Also, since this dataset is smaller than the original, I usually add a '-mini' suffix to indicate this.

```suggestion
    huggingface_id="ScandEval/jentoft-mini",
```